### DISCUSSION TITLE
Df. shape() aint working

### DISCUSSION THREAD

**Discussion by saleharashid01** [AammarTufail/python_ka_chilla2023#27] [Q&A]
im trying to execute df commads i.e. df. head(), df. tail()... but for me df. shape ain't working it is showing a screwdriver symbol (when coming in the drop-down list). kindly tell
![salp](https://user-images.githubusercontent.com/121869148/211074429-5b63540d-f954-43fc-8b2e-8fc6cc3cfd20.jpg)

**Comment by Masoodullah909** [comment]
> im trying to execute df commads i.e. df. head(), df. tail()... but for me df. shape ain't working it is showing a screwdriver symbol (when coming in the drop-down list). kindly tell ![salp](https://user-images.githubusercontent.com/121869148/211074429-5b63540d-f954-43fc-8b2e-8fc6cc3cfd20.jpg)


Remove space in between df and .shape
Since this is not a function, so remove the brackets to run the shape command
You may write: df.shape

**Reply by nouman073** [reply]
Write only  df. Shape
